sdk/storage/azqueue/service/client.go

```diff
@@ -0,0 +1,148 @@
+package service
```

Comment:
Why is this in a `service` sub-package?  Maybe a better question is, what packages are intended to be part of this module, and what purposes do they serve?

Reply:
Similar to blobs, queue has a service/account client https://learn.microsoft.com/en-us/rest/api/storageservices/operations-on-the-account--queue-service- -> this link contains operations on account level.
There will be two clients, service and queue, each with their respective sub-package.

Reply:
We did this for `azblob` due to the size/complexity of the public surface area.  My understanding is that queues is much smaller/simpler.  Do we need to separate the two?

Reply:
we separate them in other languages as well (python, Java and .NET). Technically we're already merging one extra client (the MessageClient) into QueueClient, so I think having ServiceClient and QueueClient is reasonable.

Reply:
I think it best to have both clients ServiceClient and QueueClient in a single azqueue package.